user-docs/pupil-capture.md

```diff
@@ -424,6 +437,22 @@ The exact time window to consider can be set using the `Gaze History Length` fie
 *   Streaming Surfaces with Pupil Capture - Detected surfaces as well as gaze positions relative to the surface are broadcast under the `surface` topic. Check out [this video](http://youtu.be/qHmfMxGST7A) for a demonstration.
 *   Surface Metrics with Pupil Player - if you have defined surfaces, you can generate surface visibility reports or gaze count per surface. See our [blog post](http://pupil-labs.com/blog/2014/07/0392-player-release.html) for more information.
 
+#### Legacy Markers
+
+The legacy surface system used simple square markers, which are way less robust to detect.
+For all new projects we strongly recommend using Apriltags!
+
+<aside class="warning">
+  <strong>Note</strong> - The legacy marker system has a problem in Pupil Player, where marker detection could be non-unique if the marker detection process was not started from the beginning of the recording or was discontinued while detecting by seeking in the recording. When working with legacy markers, you are strongly recommended to always only enable the offline surface detection in Pupil Player when the seek position is at frame 0 and to not seek in the video as long as the surface detection is still running, in order to avoid any data inconsistencies!
```

Comment:
This section needs to be re-worked IMO. 

```suggestion
  <strong>Note</strong> - When working with legacy markers in Pupil Player, you are strongly recommended to only enable the offline surface detection in Pupil Player when the seek position is at frame 0 and not to seek in the video as long as the surface detection is still running. Explanation: There is a bug in the legacy marker tracking system in Pupil Player. Marker detection could be non-unique if the marker detection process was not started from the beginning of the recording or was discontinued while detecting by seeking in the recording. 
```

not the best edit, but maybe could be improved by @papr or @pfaion

Reply:
I like it. It should not bee too important anyways, as users are strongly recommended to switch to apriltags for a lot of other reasons anyways. We were not even sure about whether this is a good place to include this warning.